recipe-server/normandy/conftest.py

```diff
@@ -39,4 +40,14 @@ def fake_sign(datas):
         return sigs
 
     mocked.return_value.sign_data.side_effect = fake_sign
+
+    def verify_api_pair(recipe_and_signature):
```

Comment:
I think this should be a helper on the test class instead of on the mocked class object. If it needs access to `fake_sign`, we should factor that out of this fixture.